training/coqui_stt_training/util/helpers.py

```diff
@@ -163,27 +163,43 @@ def do_iterate():
 
 
 def get_value_range(value, target_type):
+    """
+    This function converts all possible supplied values for augmentation
+    into the [start,end,r] ValueRange type. The expected inputs are of the form:
+
+    <value>
+    <value>~<r>
+    <value>:<value>~<r>
+
+    Any "missing" values are filled so that ValueRange always includes [value,value,r].
+    """
     if isinstance(value, str):
-        r = target_type(0)
-        parts = value.split('~')
-        if len(parts) == 2:
-            value = parts[0]
-            r = target_type(parts[1])
-        elif len(parts) > 2:
-            raise ValueError('Cannot parse value range')
+        if '~' in value:
+            parts = value.split('~')
+            if len(parts) == 2:
+                value = parts[0]
+                r = target_type(parts[1])
+            elif len(parts) > 2:
+                raise ValueError('Cannot parse value range')
+        else:
+            # if no <r> supplied, use 0
+            r = target_type(0)
         parts = value.split(':')
-        if len(parts) == 1:
-            parts.append(parts[0])
-        elif len(parts) > 2:
+        if len(parts) > 2:
```

Comment:
To be more explicit:
```suggestion
        if len(parts) not in (1, 2):
```

Reply:
made this more explicit with `if len(parts) != 2` after the `.append()` operation.